python/cudf/cudf/pylibcudf_tests/test_lists.py

```diff
@@ -134,3 +139,28 @@ def test_index_of_list_column(test_data, column):
     expect = pa.array(column[1], type=pa.int32())
 
     assert_column_eq(expect, res)
+
+
+def test_apply_boolean_mask(test_data, bool_column):
+    list_column = test_data[0][0]
```

Comment:
what do you think about making this one a its own fixture since it's used so much throughout the file?